### DISCUSSION TITLE
My Extensions.

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/extensions#863] [General]
Can I Add My Extensions To extensions.turbowarp.org?

Extensions:

Name - Source Code Url - Description
--------------------------
Internet - http://scratchextensions.7m.pl/extensions/internet.js - Internet extension adds 3 new blocks to interact with web. Created By [PlayerWictor](http://playerwictor.rf.gd).
YouTube - http://scratchextensions.7m.pl/extensions/youtube.js - YouTube extension adds 2 new blocks to interact with YouTube. Unofficial. Created By [PlayerWictor](http://playerwictor.rf.gd).
Search - http://scratchextensions.7m.pl/extensions/search.js - Search extension adds 1 new block to interact with Google, DDG, Yahoo Search And More. Created By [PlayerWictor](http://playerwictor.rf.gd)

**Comment by ghost** [comment]
The Internet extension has a block used in another TurboWarp extension. It is the reporter. Could you remove it?

**Comment by ghost** [comment]
Yes

**Comment by DNin01** [comment]
You can read [CONTRIBUTING.md](https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md) to learn the process, then you can submit your extensions via a pull request.

**Comment by snowboyz0825** [comment]
ngl I don't know how you'd use these, but into the archive they go
